site/docs/spec.md

```diff
@@ -1128,6 +1125,41 @@ Example
      ] } ]
 ```
 
+### Content File (Data and Delete) Serialization
```

Comment:
Not for this PR, but it would be good at some point to just include the spec from the repository itself.